miso/App.hs

```diff
@@ -0,0 +1,48 @@
+module App (start) where
+
+import Control.Monad (void)
+import Control.Monad.Trans.Maybe (MaybeT (MaybeT, runMaybeT))
+import JSDOM (currentDocument)
+import JSDOM.Generated.Node (removeChild)
+import JSDOM.Generated.NonElementParentNode (getElementById)
+import JSDOM.Generated.ParentNode (getFirstElementChild)
+import Miso hiding (getElementById)
```

Comment:
Unfortunately Miso doesn't have the cleanest API. Its own `getElementById` isn't very useful, and things like this which seem like they should be internal are for some reason exported from the top-level module.